essentials.sh

```diff
@@ -185,6 +185,23 @@ def_protected_var() {
     >&2 echo "+ ${var_name}=*****"
 }
 
+#Defines variable assigning the specified value to it if the variable does not
```

Comment:
`Defines a private variable. The function helps avoid using shellcheck disable=SC2034`.